ios/FluentUI/Command Bar/CommandBarButtonGroupView.swift

```diff
@@ -0,0 +1,59 @@
+//
+//  Copyright (c) Microsoft Corporation. All rights reserved.
+//  Licensed under the MIT License.
+//
+
+import UIKit
+
+class CommandBarButtonGroupView: UIView {
+    let buttons: [CommandBarButton]
+
+    private lazy var stackView: UIStackView = {
+        let stackView = UIStackView(arrangedSubviews: buttons)
+        stackView.translatesAutoresizingMaskIntoConstraints = false
+        stackView.axis = .horizontal
+        stackView.spacing = Constants.buttonPadding
+
+        return stackView
+    }()
+
+    init(buttons: [CommandBarButton]) {
+        self.buttons = buttons
+
+        super.init(frame: .zero)
+        translatesAutoresizingMaskIntoConstraints = false
+
+        clipsToBounds = true
+        layer.cornerRadius = Constants.cornerRadius
+        if #available(iOS 13, *) {
```

Comment:
for iPad, we should also add pointer support?

Reply:
I'm not familiar with iPad pointer. Can other contributors add this feature?

Reply:
Here's some reference PRs on Pointer Interaction support:

[https://github.com/microsoft/fluentui-apple/pulls?q=is%3Apr+pointer+in%3Atitle+](https://github.com/microsoft/fluentui-apple/pulls?q=is%3Apr+pointer+in%3Atitle+)

Reply:
I don't think it is a must. we should follow up with the feature crew?

Reply:
@rdeassis where can we track this for future work?